test/e2e/server/server_suite_test.go

```diff
@@ -0,0 +1,33 @@
+package server_test
+
+import (
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/api7/etcd-adapter/test/e2e/base"
+)
+
+var (
+	serverConfigFile, _ = filepath.Abs("../../testdata/config/server.yaml")
```

Comment:
We can use goland `embed`  feature

Reply:
The program uses `exec.Command` to start another process and set the `-c path` flag for it, and does not use the contents of the file directly, so there is no need to use go embed.

Reply:
OK